guides/source/command_line.md

```diff
@@ -458,7 +458,7 @@ $ bin/rails dbconsole --database=animals
 
 ### `bin/rails runner`
 
-`runner` runs Ruby code in the context of Rails non-interactively. For instance:
+`runner` runs Ruby code from inside the context of the Rails application non-interactively, without having to open Rails `console`. For instance:
```

Comment:
Mentioning `application` is definitely an improvement.

I'm not sure `from inside the context` is commonly used:
https://trends.google.com/trends/explore?date=now%201-d&geo=NL&q=%22inside%20the%20context%20of%22,%22in%20the%20context%22&hl=en-US

```suggestion
`runner` runs Ruby code in the context of the Rails application non-interactively, without having to open Rails `console`. For instance:
```

The following is used in the `runner` help command:
https://github.com/rails/rails/blob/6a9795d4d22d7bf608bf89fb8b08c1307c5d0d57/railties/lib/rails/commands/runner/USAGE#L2